recipes/catch2/2.x.x/conanfile.py

```diff
@@ -14,10 +14,21 @@ class ConanRecipe(ConanFile):
     exports_sources = ["CMakeLists.txt", "patches/**"]
     generators = "cmake"
     settings = "os", "compiler", "build_type", "arch"
-    options = {"fPIC": [True, False], "with_main": [True, False]}
-    default_options = {"fPIC": True, "with_main": False}
+    options = {
+        "fPIC": [True, False],
+        "with_main": [True, False],
+        "with_benchmark": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+        "with_main": False,
+        "with_benchmark": False,
+    }
     _cmake = None
 
+    def validate(self):
+        if tools.Version(self.version) < "2.13.1" and self.settings.arch == "armv8":
+            raise ConanInvalidConfiguration("ARMv8 is supported by 2.13.1+ only! give up!")
```

Comment:
Can you move this code to the `configure` method below,
and rename `configure` to `validate`?

Then the order of methods is fully compatible with other recipes. :)